### PR TITLE
Users CRUD

### PR DESCRIPTION
**Pull request by kingobarn**
Updated admin page with following features:

Admin toggle button
Appplication status
Approve/Reject content creator application
Delete user
Adjust rows per page
General UI overhaul to replicate the one in figma

Below is a video showing all features (ignore my opened tabs pls):

https://github.com/user-attachments/assets/7a0bee24-3c08-4961-a5df-c2acb2a83082

### CODE REVIEW

**Comment by LazarShockX** [comment]
There's a syntax error in Navbar.tsx after we merged, please standby 🫡

**Review by LucaPurcilly — APPROVED** [review]
Looks good!

**Comment by madsWesth** [comment]
Minor thing, but you should delete the `.idea` folder and add it to `.gitignore`

File: app/src/services/auth.services.ts [hunk 1]
```diff
@@ -37,8 +38,8 @@ const AcceptApplication = async (id: string): Promise<unknown> => {
   return await axios.put(`${BACKEND_URL}/api/applications/${id}approve`);
 };
 
-const RejectApplication = async (id: string, reason: string): Promise<unknown> => {
-  return await axios.put(`${BACKEND_URL}/api/applications/${id}/reject`, { reason });
+const RejectApplication = async (id: string, rejectionReason: string): Promise<unknown> => {
```
[the remark this answers is not in the document]
Comment (LazarShockX) [reply app/src/services/auth.services.ts:41]: We decided not to move it to Admin

File: app/src/components/DeleteUserButton.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  applicationId: string;
+  onDelete: () => void;
+}
+
+const DeleteUserButton: React.FC<DeleteUserButtonProps> = ({ applicationId, onDelete }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  const handleDelete = async () => {
+    try {
+      console.log("Deleting user with ID:", applicationId);
```
Comment (madsWesth) [inline app/src/components/DeleteUserButton.tsx:18]: remove console.logs

File: app/src/pages/ViewUserButton.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      const userDetails = await AdminServices.getSingleUserDetails(applicationId, token);
+      console.log("User Details: ", userDetails);
+      setUserDetails(userDetails);
+
+      const userApplication = await AuthServices.GetSingleCCApplication(applicationId);
+      console.log("User Application: ", userApplication);
+      setUserApplication(userApplication.data);
+
+      const contentCreator = await AdminServices.getContentCreator(applicationId, token);
+      console.log("Content Creator: ", contentCreator);
```
Comment (madsWesth) [inline app/src/pages/ViewUserButton.tsx:34]: more console.logs to delete

**Review by madsWesth — COMMENTED** [review]
minor things to remove but other wise LGTM 👍

File: app/src/components/DeleteUserButton.tsx (hunk 2 above)
Comment (LazarShockX) [reply app/src/components/DeleteUserButton.tsx]: Done

File: app/src/pages/ViewUserButton.tsx (hunk 3 above)
Comment (LazarShockX) [reply app/src/pages/ViewUserButton.tsx]: Done

**Pull request merged**